docs/miairr/specification_miairr_ncbi.rst

```diff
@@ -193,24 +194,40 @@ The header MUST contain all of the following elements:
 -  REQUIRED: header structure as specified by the respective INSDC
    repository [ENA_MANUAL]_ [GENBANK_FF]_ [GENBANK_SR]_.
 
+-  FORBIDDEN: The ``DEFINITION`` entry will be autopopulated by
+   information provided in the FT part (``misc_feature``, ``/note``).
+
 -  REQUIRED: identifier of the associated SRA record (MiAIRR data
-   set 4) as ``DR`` line (GenBank: ``DBLINK``). Note that it is **not**
+   set 4) as ``DBLINK`` (ENA: ``DR`` line). Note that it is **not**
    possible to refer to individual raw reads, only the full SRA
    collections can be linked.
 
--  REQUIRED: the term "AIRR" is included in the ``KW`` line (GenBank:
-   ``KEYWORDS``).
+-  REQUIRED: in the ``KEYWORDS`` field (ENA: ``KW`` line):
+
+   -  the term "TLS"
+
+   -  the term "Targeted Locus Study"
+
+   -  the term "AIRR"
+
+   -  the term "MiAIRR:<x>.<y>" with <x> and <y> indicating the used
+      version and subversion of the MiAIRR standard.
 
 -  REQUIRED: DOI of the associated free-text record containing the
-   information on data processing (MiAIRR data set 5) as ``RX`` line
-   (GenBank: ``REMARK`` within a ``REFERENCE`` [4]_.
+   information on data processing (MiAIRR data set 5) as ``REMARK``
+   within a ``REFERENCE`` [4]_ (ENA: ``RX`` line).
+
+-  OPTIONAL: The use of `structured records`_ is currently evalutated
```

Comment:
`structured comments` is currently being evaluated

Reply:
Fixed.